composer.json

```diff
@@ -52,23 +54,5 @@
             "email": "kehinde.ladipo@cottacush.com",
             "role": "Software Engineer"
         }
-    ],
```

Comment:
@adaugochi Why are you removing this?

Reply:
It has deprecated, I am making use of `fxp` instead